client-java/controller/src/main/java/org/evomaster/client/java/controller/problem/rpc/CodeJavaGenerator.java

```diff
@@ -303,6 +303,15 @@ public static String junitAssertEquals(String expectedValue, String variableName
         if (AssertionsUtil.getAssertionsWithComment(assertionScript)){
             return "//" + assertionScript;
         }
+
+        /*
+            if length of expected value is more than 10, comment out this assertion for the moment
+
+            TODO need to better handle flakiness later
+         */
+        if (expectedValue.length() > 10)
```

Comment:
hmmmm. not sure i like this. but i guess it is fine for the moment as it applied only for RPC. but please create a Trello card mentioning this file and TODO

Reply:
this now is clarified in this card, https://trello.com/c/v5XRNyml/707-flakiness-configurable-fields-to-skip